### ISSUE TITLE
Project deleted after using root path as docs dir in configuration

### ISSUE THREAD

**Issue by juanmanuelramallo** [zipmark/rspec_api_documentation#441]
It would be nice at least to mention in the readme that the `docs dir` should be a folder exclusively for docs, cause all its content will get erased after generating docs. 

I just lost all my local files after running the docs generator using Rails.root as the docs dir.

**Comment by jakehow** [comment]
@juanmanuelramallo sorry this happened to you.  Did you read the whole README before running the rake task or would this have happened to you anyway?

We could prompt on first run or always prompt unless you set an ENV that might make this more unlikely?

**Comment by juanmanuelramallo** [comment]
Thanks for answering. 

First of all, excellent work. I just generated my api-blueprint successfully (apiary likes it 🙈) and I wanted to thank you for that. Really good work.

Now, while reading the readme I came across with the configuration option to specify a different docs directory. Then I thought it would be nice to generate the docs just in the root folder (as I only want a *.apib in the root folder for the docs renderer to work). 

So I changed it. Then ran again the task and everything was gone (like some Thano's clicking 😂).

Went inside the repo and found this https://github.com/zipmark/rspec_api_documentation/blob/master/lib/rspec_api_documentation/writers/writer.rb#L18 which seems ok if we have a specific folder for the generated documentation. 

My suggestion is to specify in the readme that the option `docs_dir` needs to be a folder specifically for the docs. Cause its content gets removed every time the docs are generated.

Another option would be to (somehow) blacklist the paths that could be harmful for the project if gets removed, like:
- Rails.root
- app/
- config/
- ...
And raise a runtime error if it happens.

Anyways, thanks for your time! I really appreciate your work on this.
